Makefile

```
@@ -119,6 +119,28 @@ e2e-test-clean-crds: ## Delete all scaled objects and jobs across all namespaces
 e2e-test-clean: get-cluster-context ## Delete all namespaces labeled with type=e2e
 	kubectl delete ns -l type=e2e
 
+# The OpenShift tests are split into 3 targets because when we test the CMA operator, 
+# we want to do the setup and cleanup with the operator, but still run the test suite
+# from the test image, so we need that granularity. 
+.PHONY: e2e-test-openshift-setup
+e2e-test-openshift-setup: ## Setup the tests for OpenShift
+	@echo "--- Performing Setup ---"
+	cd tests; go test -v -timeout 15m -tags e2e ./utils/setup_test.go 
```

```suggestion
	go test -v -timeout 15m -tags e2e ./tests/utils/setup_test.go 
```
Is that not possible? I am sure I am missing some requirement of `go test`.

---

It is possible, but their entire test doc runs everything out of that tests directory so I thought there was a % chance it might care or make assumptions that's how it was being done, so I left it alone: https://github.com/kedacore/keda/tree/main/tests#running-tests